packages/core/src/themes/pentahoPlus.ts

```diff
@@ -208,10 +208,19 @@ export const pentahoPlus = mergeTheme(pentahoPlusBase, {
         tagsList: {
           backgroundColor: inputColors.bg,
           padding: theme.space.xxs,
+          borderColor: theme.colors.textDimmed,
         },
         singleLine: {
           height: 32,
         },
+        disabled: {
+          "& .HvTagsInput-tagsList": {
+            backgroundColor: theme.colors.bgDisabled,
+            "&,:hover": {
+              borderColor: theme.colors.textDisabled,
+            },
+          },
```

Comment:
Can we also add the Tag border, so that it looks like the spec? 🙏 
```suggestion
          },
          "& .HvTagsInput-chipRoot": {
            outlineColor: theme.colors.textDisabled,
          },
```